bin/docker_connection.py

```diff
@@ -45,6 +45,9 @@ async def _start_docker_controller(docker_pipe: aioprocessing.AioConnection):
             await asyncio.sleep(0.5)
             continue
 
+        if initialization_error is not None:
+            await docker_pipe.coro_send(initialization_error)
```

Comment:
You should add a "continue" after this s.t. it doesn't try to call the docker client and generate more exceptions.